lua/telescope/utils.lua

```diff
@@ -216,13 +216,25 @@ end)()
 utils.path_tail = (function()
   local os_sep = utils.get_separator()
 
-  return function(path)
-    for i = #path, 1, -1 do
-      if path:sub(i, i) == os_sep then
-        return path:sub(i + 1, -1)
+  if os_sep == "/" then
```

Comment:
`vim.fn.fnamemodify(path, ":p:t")` may replace `util.path_tail(path)`, but the former may be slower and has dependency on neovim